src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -117,13 +120,15 @@ private DownloadMapsWindow(
       progressPanel.download(pendingDownloads);
     }
 
+    pendingDownloads.addAll(ClientContext.DOWNLOAD_COORDINATOR.getDownloads().stream()
```

Comment:
Do "terminated" and "cancelled" mean the same thing in this context?  It looks that way from the code, but I want to make sure I'm not misunderstanding something subtle since I'm half asleep. :smile: